### PR TITLE
Activity 3 - Online Resume of Emann Arthur D. Capunong

### PR DESCRIPTION
**Pull request by thormaximus826**

### CODE REVIEW

File: Capunong_Emann Arthur/05-12-2.html
Comment (mrmartincarlito) [inline Capunong_Emann Arthur/05-12-2.html:1]: please remove this file

Comment (mrmartincarlito) [reply Capunong_Emann Arthur/05-12-2.html:1]: HI @thormaximus826 please remove all your practice html files in your activity folder

Comment (thormaximus826) [reply Capunong_Emann Arthur/05-12-2.html:1]: Copy

Comment (thormaximus826) [reply Capunong_Emann Arthur/05-12-2.html:1]: Done SIr

File: Capunong_Emann Arthur/htmlactivity.html
Comment (mrmartincarlito) [inline Capunong_Emann Arthur/htmlactivity.html:1]: remove this also

File: Capunong_Emann Arthur/htmlpractice.html
Comment (mrmartincarlito) [inline Capunong_Emann Arthur/htmlpractice.html:1]: this also

File: Capunong_Emann Arthur/practicehtml.html
Comment (mrmartincarlito) [inline Capunong_Emann Arthur/practicehtml.html:1]: this also

File: Capunong_Emann Arthur/05-12-2.html
Comment (mrmartincarlito) [reply Capunong_Emann Arthur/05-12-2.html:1]: some files are still existing
kindly retain activity3 only

Comment (thormaximus826) [reply Capunong_Emann Arthur/05-12-2.html:1]: Done sir

File: Capunong_Emann Arthur/htmlactivity.html
Comment (thormaximus826) [reply Capunong_Emann Arthur/htmlactivity.html:1]: Done Sir

File: Capunong_Emann Arthur/htmlpractice.html
Comment (thormaximus826) [reply Capunong_Emann Arthur/htmlpractice.html:1]: Done Sir

File: Capunong_Emann Arthur/practicehtml.html
Comment (thormaximus826) [reply Capunong_Emann Arthur/practicehtml.html:1]: Done Sir

**Pull request merged**